### PR TITLE
Hw8

### PR DESCRIPTION
**Pull request by ellinhoms9**
HW8 is done

### CODE REVIEW

File: HW8/task3.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+def string_elem_to_up(elem):
+    print(f'Argument of function: {elem}')
+    if type(elem) == str:
```
Comment (acman) [inline HW8/task3.py]: nit: nice function for checking type is `isinstance()`

File: HW8/task4.py [hunk 2]
```diff
@@ -0,0 +1,10 @@
+def filter_int(elem):
+    # check type of elements
+    if type(elem) == int or type(elem) == float:
+        return elem
```
Comment (acman) [inline HW8/task4.py]: not correct here it returns an element of the sequence and `filter` check if it is bool, so 0 element will be omitted. So boolean value should be returned
```suggestion
    return isinstance(elem, (int, float))
```

**Review by acman — CHANGES_REQUESTED** [review]
Overall looks good but filter function should be corrected

File: HW8/task4.py (hunk 2 above)
Comment (ellinhoms9) [reply HW8/task4.py]: i am changed this mistake

File: HW8/task3.py (hunk 1 above)
Comment (ellinhoms9) [reply HW8/task3.py]: i am changed this mistake

**Review by acman — APPROVED** [review]
Nice job!

**Pull request merged**